pkg/config/config.go

```diff
@@ -94,6 +95,13 @@ type OverlayOptionsConfig struct {
 	Size string `toml:"size"`
 	// Do not create a bind mount on the storage home
 	SkipMountHome string `toml:"skip_mount_home"`
+	// ForceMask indicates a mask of permissions (e.g. "0755") that are always set for
+	// created files.  The file permissions will be: ORIGINAL_PERMISSIONS | FORCE_MASK.
+	ForceMask string `toml:"force_mask"`
```

Comment:
for the NFS case, the mask to use is `0700` so they won't become world readable (unless they were already so that ORing the original permission with 0700 won't change the group and world permissions).

The `ForceMask=0755` is a different use case.  That is useful to share the storage with other users.  In that cases all the files/directories must be world readable and accessible.

Do you think we should make this distinction clearer somehow?

Reply:
I’m not disputing the 0700/0755 values. But consider that the default permissions of `/` in a container are 0555, so even in the “private” setting, the file will end up with external 0755 (and an in-container 0777 file would end up 0777 external regardless of `ForceMask`). Is that OK, or should the file really end up _exactly_ 0700/0755?

Reply:
I am not sure.  Both ways are enough for the NFS storage and shared storage cases.

I thought having the `ForceMask` work as an "opposite" umask could give more flexibility in the future, but at the moment I cannot think of any other use case where it would be useful

Reply:
I've changed the semantic to mean "final permissions mask".  I guess it is easier to understand and also avoids some issues if a file in an image is world writeable and we end up opening the image to every other user.